tests/nodes/NodeConnectionManager.general.test.ts

```diff
@@ -76,10 +77,10 @@ describe(`${NodeConnectionManager.name} general test`, () => {
 
   let dataDir: string;
 
-  let remotePolykeyAgent: PolykeyAgent;
+  let remotePolykeyAgentA: PolykeyAgent;
```

Comment:
Creating an agent is comparatively expensive. I should check where they're being used and reduce the amount of times I need to start an agent.